
target_frame: velodyne
# target_frame: camera_link # Leave disabled to output scan in pointcloud frame
transform_tolerance: 0.01
min_height: -0.6
max_height: 1.0

angle_min: -3.14 # -M_PI
angle_max: 3.14 # M_PI
angle_increment: 0.0087 # M_PI/360.0
scan_time: 0.3333
range_min: 0.45
range_max: 20.0
use_inf: true
inf_epsilon: 1.0

# Concurrency level, affects number of pointclouds queued for processing, thread number governed by nodelet manager
# 0 : Detect number of cores
# 1 : Single threaded
# 2->inf : Parallelism level
concurrency_level: 1
